### DISCUSSION TITLE
VPN tunnel is established, but we cannot ping between each other.

### DISCUSSION THREAD

**Discussion by jbenavidescr** [strongswan/strongswan#1964] [General]
i'm having an issue with a site to site tunnel, i think its iptables related, anyone can give me a hand here? i will appreciate it. 

https://pastebin.com/GPkAC3yf

**Comment by tobiasbrunner** [comment]
Could be a NAT issue. See the [docs on forwarding traffic](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html).

**Reply by jbenavidescr** [reply]
I have added the following rule on the gateway and now I can ping from gateway to a host on the other side.

**_sudo iptables -t nat -A POSTROUTING -o enp9s0 -j MASQUERADE_**

But i cannot ping from a host on left side to a host on right side.

Host on right side CAN ping to a host on left side.

Something I am missing on the gateway config, i have tried everything in https://docs.strongswan.org/docs/5.9/howtos/forwarding.html

but no luck at all.

 any other ideas?

**Reply by tobiasbrunner** [reply]
You obviously can't ping an individual client if you NAT their client IPs (presumably virtual, but without config we don't know) to the gateway's IP. So please reconsider whether NAT or routing should be used (refer to the document for the different options).
